samples/app-dev/API-Development/REST/allOf_oneOf-anyOf_construct_schema/readme.md

```diff
@@ -43,24 +40,23 @@ Rest triggers in the service app have been configured with Open API Specs 3.0 fi
 
 
 ## Understanding the configuration
-After importing the 'AllOf_OneOf_AnyOf_Keywords_APISpec_Service', we can see that multiple response codes have been configured in the Rest trigger. Few Response codes have Response headers as well. While adding schema in Response header and Response body we need to make sure that both are in JSON data or JSON Schema format.
-![MultipleResponseCode_configuration](./import-screenshots/MultipleResponse_serviceApp.png)
+In *Invoke_AllOf_OneOf_AnyOf_Keywords_APISpec_Service* app we have three flows *Invoke_AnyOfKeywordAPISpecConfiguredFlow*, *Invoke_AllofKeywordAPISpecConfiguredFlow* and *Invoke_OneOfKeywordAPISpecConfiguredFlow* for anyOf, allOf and oneOf keyword.
+![anyOf_keyword_response_body](./import-screenshots/anyOf_response_body.png)
+![allOf_keyword_response_body](./import-screenshots/allOf_response_body.png)
+![oneOf_keyword_response_body](./import-screenshots/oneOf_response_body.png)
 
-ConfigureHTTPResponse activity should be used when we have configured multiple response codes in the Rest trigger. This activity is useful in mapping Response body and Response headers of a particular Response code in getting input from other activities and output to 'Return' activity.
-![ConfigureHTTPResponse_Input](./import-screenshots/ConfigureHTTPResponse_Input_Mappings.png)
-
-For the 'Invoke_AllOf_OneOf_AnyOf_Keywords_APISpec_Service' app, we have configured the activity with the API Spec of the 'AllOf_OneOf_AnyOf_Keywords_APISpec_Service'. We can upload swagger 2.0 or Open API spec 3.0 in the InvokeRestService activity to configure it. Request parameters, Request schema, Response schema and header would be auto-populated when a valid API spec file is uploaded. After uploading the spec we just need to map the input.
+For the 'Invoke_AllOf_OneOf_AnyOf_Keywords_APISpec_Service' app, we have configured the activity with the API Spec of the 'AllOf_OneOf_AnyOf_Keywords_APISpec_Service'. We have to upload Open API spec 3.0 in the InvokeRestService activity to configure it. After uploading the spec we just need to map the input.
```

Comment:
We've uploaded three OAS specs for each of the service not one API spec of AllOf_OneOf_AnyOf_Keywords_APISpec_Service, please mention which API spec is uploaded in which flow InvokeRest activity.
No need to repeat "We have to upload Open API spec 3.0 in the InvokeRestService activity to configure it"
We can remove this also "After uploading the spec we just need to map the input." here